src/internal/connector/exchange/api/mail.go

```diff
@@ -286,16 +286,11 @@ func (c Mail) GetAddedAndRemovedItemIDs(
 	pgr := &mailPager{service, builder, options}
 
 	if len(os.Getenv("CORSO_URL_LOGGING")) > 0 {
-		gri, err := builder.ToGetRequestInformation(ctx, options)
+		gri, err := builder.ToGetRequestInformation(ctx, nil)
```

Comment:
Is there a reason why the `options` are not being passed in?

Reply:
copy-paste error, thanks